python/cudf/cudf/tests/test_string.py

```diff
@@ -1828,3 +1828,31 @@ def test_string_str_decode_url(data):
     got = gs.str.url_decode()
     expected = pd.Series([urllib.parse.unquote(url) for url in data])
     assert_eq(expected, got)
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        ["f0:18:98:22:c2:e4", "00:00:00:00:00:00", "ff:ff:ff:ff:ff:ff"],
+        ["f0189822c2e4", "000000000000", "ffffffffffff"],
+        # ['0xf0189822c2e4', '0x000000000000', '0xffffffffffff'],
+        # ['0Xf0189822c2e4', '0X000000000000', '0Xffffffffffff'],
```

Comment:
Cleanup comments?

Reply:
This needs to be supported and for which I have created #5113 , once it is fixed, we can use these cases as well.

Reply:
Ah, Cool @rgsl888prabhu 👍